### PR TITLE
Update slang.md

### PR DESCRIPTION
**Pull request by tuxgoose**
There are numerous  missing slang terms in the dictionary

I expect to have the additional  key terms in our slang dictionary

### CODE REVIEW

File: slang.md [hunk 1]
```diff
@@ -13,3 +13,9 @@ Former functional human beings that suffered from the Nueroinfinitus (forevermor
 ### Forevermore Plague
 
 Nueroinfinitus also known as the Forevermore Plague was a widedspread virus that effected nearly all persons who had cyber implants
+
+### The Road
+The YANG LIWEI INTERNATIONAL SPACE LIFT. nothing on earth has caused a larger technological, scientific, economic, or industrial boom. Some say "The Road" is the apex achievement of humankind.
+
+### E-Sheks / Joe-Bucks / Script
```
Comment (tuxgoose) [inline slang.md:20]: Joe Buck? Was this a real person? @The-Real-Matthew-Dycus

File: slang.md [hunk 2]
```diff
@@ -13,3 +13,9 @@ Former functional human beings that suffered from the Nueroinfinitus (forevermor
 ### Forevermore Plague
 
 Nueroinfinitus also known as the Forevermore Plague was a widedspread virus that effected nearly all persons who had cyber implants
+
+### The Road
+The YANG LIWEI INTERNATIONAL SPACE LIFT. nothing on earth has caused a larger technological, scientific, economic, or industrial boom. Some say "The Road" is the apex achievement of humankind.
```
Comment (tuxgoose) [inline slang.md]: I like the bold title for the space lift, but should we format this differently?

**Review by tuxgoose — COMMENTED** [review]
Looks good, I feel we could add a few more key terms now that the zine is out.

File: slang.md (hunk 1 above)
Comment (The-Real-Matthew-Dycus) [reply slang.md:20]: yes! Joseph Stalin :) its the new ruble

File: slang.md (hunk 2 above)
Comment (The-Real-Matthew-Dycus) [reply slang.md]: i just copied and pasted from roll 20-  yeah it should be lowercase

Comment (The-Real-Matthew-Dycus) [reply slang.md:18]: yep! i just copy/pasted from roll 20 it should be consistent here ill make lowercase

File: slang.md (hunk 1 above)
Comment (The-Real-Matthew-Dycus) [reply slang.md:20]: yes, joe bucks are based on joseph stalin ,

File: slang.md (hunk 2 above)
Comment (tuxgoose) [reply slang.md]: @The-Real-Matthew-Dycus  I will modify to lower case.

**Comment by tuxgoose** [comment]
Hey everything looks goo

**Pull request merged**